### DISCUSSION TITLE
Postal is not sending emails - email status is pending

### DISCUSSION THREAD

**Discussion by markmoroni25** [postalserver/postal#2235] [Help with using Postal]
### Discussed in https://github.com/postalserver/postal/discussions/1726

<div type='discussions-op-text'>

<sup>Originally posted by **speedeonic** November 23, 2021</sup>
Hello,

I am new to Postal mail server and I need help on this problem or bug. 
It may also be helpful for others who face this problem.

I have installed Postal on Ubuntu 18.04 (64 Bit) and Docker.
I followed the instructions by a video tutorial on YouTube and installed everything like Docker compose, Maria Db, RabbitMQ etc. I setup all the required dns and finally Postal got installed and it seemed working.

But when I tried to send emails from newly installed Postal, emails didn't deliver and the email status is "Pending".

I checked the Postal status and found these values.

postal_cron_1       /docker-entrypoint.sh post ...   Up
postal_requeuer_1   /docker-entrypoint.sh post ...   Exit 1
postal_smtp_1       /docker-entrypoint.sh post ...   Up
postal_web_1        /docker-entrypoint.sh post ...   Up
postal_worker_1     /docker-entrypoint.sh post ...   Exit 1

Can anyone please help me and guide me on how to fix it?

Thank you
Faiz</div>

**Comment by willpower232** [comment]
As per the discussion you already linked to, you need to look at the logs to see why things aren't working.

**Reply by markmoroni25** [reply]
Hi Will, 

How do you look at the logs?

**Reply by willpower232** [reply]
You should be able to see them with `postal logs worker` and `postal logs requeuer`
